plutus-tx/src/PlutusTx/Prelude.hs

```diff
@@ -181,83 +202,69 @@ import PlutusTx.Traversable as Traversable
 
 import Prelude qualified as Haskell (return, (=<<), (>>), (>>=))
 
--- this module does lots of weird stuff deliberately
-{- HLINT ignore -}
-
--- $prelude
--- The PlutusTx Prelude is a replacement for the Haskell Prelude that works
--- better with Plutus Tx. You should use it if you're writing code that
--- will be compiled with the Plutus Tx compiler.
--- @
---     {-# LANGUAGE NoImplicitPrelude #-}
```

Comment:
Where did this comment go?

Reply:
Fixed this as well. good catch, thanks

Reply:
I still don't see this comment block with PlutusTx.Prelude description in your PR, looks like it got erased.

Reply:
Stylish haskell erases this :upside_down_face: It should be added back.